mix.exs

```diff
@@ -16,16 +16,25 @@ defmodule Jellyfish.MixProject do
       deps: deps(),
       dialyzer: dialyzer(),
 
-
-            # hex
-            description: "Template Plugin for Membrane Multimedia Framework",
-            package: package(),
-
-            # docs
-            name: "Membrane Template plugin",
-            source_url: @github_url,
-            homepage_url: "https://membraneframework.org",
-            docs: docs()
+      # hex
+      description: "Jellyfish media server",
+      package: package(),
+
+      # docs
+      name: "Jellyfish media server",
+      source_url: @github_url,
+      homepage_url: "https://membrane.stream",
+      docs: docs(),
```

Comment:
It is very unlikely that we will use `ex_doc` for documentation, could we remove it?